tests/AllSolidityFeatures/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1098,11 +1106,21 @@ uint256 constant MULTIPLIER = 2**EXPONENT;
 
 contract WithUncheckedBlock {
     function f() public pure returns (uint256) {
+        uint256 x = 0;
+        unchecked {x--;}
+        return x;
+    }
+
+    function g() public pure returns (uint256) {
         uint256 x = 0;
```

Comment:
qq I'm noticing we are also changing the type of this variable, since when are we doing this?

Reply:
Hey,
This is the `explicit-types` option. We have been doing this since `v1.0.0-alpha-27`

Reply:
In #461 you can see that the bytecode doesn't change, even if the AST slightly changes.